app/static/js/NeighbourPlotCanvas.js

```diff
@@ -1,6 +1,37 @@
 var _ = require('lodash');
 var d3 = require('d3');
+
 var config = require('../config/plots').neighbourPlot;
+
+// browserify can't load modules dynamically due to the way it statically
+// analyses paths when creating the output bundle. in other words, no variables
+// or concatenated strings can be used in required paths.
+// this switch is inelegant and should be replaced with something nicer
+// maybe this could be configured server-side maybe when matplotlib 1.5 is
+// released and these colour scales become available??
```

Comment:
Huh? mpl 1.5 is available both in PyPI and conda. But how does this help in js? At any rate I don't find the switch statement *that* offensive!